excalidraw-app/components/PostHogProvider.tsx

```diff
@@ -0,0 +1,27 @@
+import { PostHogProvider } from "posthog-js/react";
+
+interface PostHogProviderWrapperProps {
+  children: React.ReactNode;
+}
+
+const PostHogProviderWrapper: React.FC<PostHogProviderWrapperProps> = ({
+  children,
+}: PostHogProviderWrapperProps) => {
+  const apiKey = import.meta.env.VITE_PUBLIC_POSTHOG_KEY;
```

Comment:
The variables names should be VITE_APP_POSTHOG_KEY VITE_APP_POSTHOG_HOST to be consistent with the other variables of the excalidraw project i think.